### PR TITLE
[15.0][MIG] survey_result_mail: Migration to 15.0

### PR DESCRIPTION
**Pull request by chienandalu**
Migrating from #74 

cc @Tecnativa TT44285

please review @pedrobaeza @stefan-tecnativa

### CODE REVIEW

File: survey_result_mail/reports/survey_answer_report.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+                <t
+                    t-call="survey_result_mail.survey_result_report_view_document"
+                    t-lang="user_input.partner_id.lang or user_input.env.lang"
+                />
+            </t>
+        </t>
+    </template>
+    <record id="survey_result_report" model="ir.actions.report">
+        <field name="model">survey.user_input</field>
+        <field name="report_name">Survey results</field>
```
Comment (stefan-tecnativa) [inline survey_result_mail/reports/survey_answer_report.xml]:
```suggestion
        <field name="name">Survey results</field>
```

File: survey_result_mail/readme/CONFIGURE.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+To configure automatic result sending to the user:
+
+#. Go to the survey you want to configure.
+#. In the *Options* tab, you'll find a new *Send survey answers* option that you can set
```
Comment (stefan-tecnativa) [inline survey_result_mail/readme/CONFIGURE.rst:4]:
```suggestion
#. In the *Options* tab, you'll find a new *Send Result Mail* option that you can set
```
This is how it appears in odoo

**Comment by stefan-tecnativa** [comment]
Functionally tested. LGTM 👍️

File: survey_result_mail/reports/survey_answer_report.xml (hunk 1 above)
Comment (chienandalu) [reply survey_result_mail/reports/survey_answer_report.xml]: Oh, thanks :+1:

File: survey_result_mail/readme/CONFIGURE.rst (hunk 2 above)
Comment (chienandalu) [reply survey_result_mail/readme/CONFIGURE.rst:4]: I change the field string better

**Comment by chienandalu** [comment]
please review @pilarvargas-tecnativa

**Review by pilarvargas-tecnativa — APPROVED** [review]
thanks :)

**Review by pedrobaeza — APPROVED** [review]
/ocabot migration survery_result_mail
/ocabot merge nobump

**Pull request merged**